### PR TITLE
Databasefix

### PR DESCRIPTION
**Pull request by egedebruin**
@bvanwijk and I fixed some bugs in the database, could you double check there are no errors anymore in the database? Otherwise I think this can be merged.

### CODE REVIEW

File: backend/configs/main.conf [hunk 1]
```diff
@@ -17,4 +17,6 @@ camera_4_address = 192.168.10.101
 serverip=localhost
 serverport=8888
 sqluser=root
-sqlpassword=root
\ No newline at end of file
+sqlpassword=root
```
Comment (Kingdorian) [inline backend/configs/main.conf]: Maybe we should pass these as command line parameters when starting the servers (maybe keep these as default if they're not provided in the command line).

**Comment by Kingdorian** [comment]
Looks good!

File: backend/src/test/java/com/benine/backend/PresetControllerTest.java [hunk 2]
```diff
@@ -43,6 +43,20 @@ public void testAddPreset() throws Exception {
   }
   
   @Test
+  public void testUpdatePreset() throws Exception {
```
Comment (jochem725) [inline backend/src/test/java/com/benine/backend/PresetControllerTest.java]: I think most of the code in these tests can be put in an `@Before` statement. It cleans up a lot of duplicate code in the tests.

Comment (jochem725) [inline backend/src/test/java/com/benine/backend/PresetControllerTest.java]: I think most of the code in these tests can be put in an `@Before` statement. It cleans up a lot of duplicate code in the tests.

**Comment by jochem725** [comment]
I've placed a small comment at a test, after it is fixed I think this can be merged.

**Comment by NaomidR** [comment]
Looks good, I do agree with the comments above though. There are no checkstyle/pmd/findbugs warnings so after fixing the above I agree with merging.

**Comment by jochem725** [comment]
Great! I'll merge this.

**Pull request merged**